bears/c_languages/CPPCheckBear.py

```diff
@@ -0,0 +1,35 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable="cppcheck",
+        use_stdout=False,
+        use_stderr=True,
+        output_format="regex",
+        output_regex=r'(?P<line>\d+):(?P<severity>[a-zA-Z]+):'
+                     r'(?P<origin>[a-zA-Z]+):(?P<message>.*)',
+        severity_map={"error": RESULT_SEVERITY.MAJOR,
+                      "warning": RESULT_SEVERITY.NORMAL,
+                      "style": RESULT_SEVERITY.INFO})
+class CPPCheckBear:
+    """
+    Report possible security weaknesses for C/C++.
+    For more information, consult <https://github.com/danmar/cppcheck>.
+    """
+
+    LANGUAGES = "C", "C++"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         enable_all_checks: bool=True):
+        """
+        :param enable_all_checks:
+            Report warnings and style issues too.
+        """
+        args = ("--template='{line}:{severity}:{id}:{message}'",)
```

Comment:
the quotes are not needed after `template=`

instead they are not escaped and the message appears with a trailing `'`.

```
bad.c
|   6| ••••f(100);
|    | [MAJOR] CPPCheckBear (arrayIndexOutOfBounds):
|    | Array 'p[10]' accessed at index 100, which is out of bounds.'
|    | The following actions are applicable to this result:
|    |  0: Apply no further actions.
|    |  1: Open the affected file(s) in an editor.
|    | Please enter the number of the action you want to execute (Ctrl-D to exit). Program terminated by user.
```